pkgs/misc/ite-backlight/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, pkgs
+, stdenv
+, ninja
+, libusb
+, meson
+, boost
+, fetchFromGitHub
+, pkg-config
+, cpp-gsl }:
+
+stdenv.mkDerivation rec {
+  pname = "ite-backlight";
+  version = "v1.1";
+
+  src = fetchFromGitHub {
+    owner = "hexagonal-sun";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256:1hany4bn93mac9qyz97r1l858d48zdvvmn3mabzr3441ivqr9j0a";
+  };
+
+  nativeBuildInputs = [
+    ninja
+    pkg-config
+    meson
+    cpp-gsl
+    boost
+  ];
+
+  buildInputs = [
+    libusb
```

`libusb` is an alias of `libusb1`.

```suggestion
    libusb1
```